tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,39 @@
 """
 
 
+test_file = """
+#include "CPPCleanBearHeaderTest1.h"
+#include "CPPCleanBearHeaderTest2.h"
+
+int main() {
+    cppcleanbearfunction1();
+    cppcleanbearfunction2();
+    return 0;
+}
+"""
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
```

Comment:
unnecessary change, if you like to have it please separate commit :+1:

Reply:
@Makman2 I added the tempfile_kwargs to ensure that the file is .cpp. I could remove that should i?

Reply:
it's okay, as cppclean works on cpp files, but it isn't necessary to be inside this commit (it did work before)

Reply:
Ok I will remove it :) Also appveyor is failing due to an XMLBearTest, that is some other issue right? not related to this PR?

Reply:
yes not related, I've created a PR that fixes / suppresses the error. As long as nothing else fails, your PR is fine :+1: we won't merge it until the fix is on master, but after a rebase your build should go immediately green :+1:

Reply:
Ok I am working on improving my PR, one other thing I experienced travis and circle failing too because of some sourceforge ssh connection issue.

Reply:
hopefully that just goes away :3 timeouts occur quite often in coala as we install many dependencies :+1:

Reply:
They failed :( same error.

Reply:
They did fail again with the same error. Seems some issue with the sourceforge link i guess 👍